compliance/src/main/java/bio/terra/pearl/compliance/SyncVantaUsers.java

```diff
@@ -380,6 +388,7 @@ private void setInScope(String accessTokeen, Collection<VantaObject> vantaObject
         try {
             String updateResult = getWebClientForIntegration(accessTokeen, integrationId, resourceKind)
                     .patch().bodyValue(updateMetadata).retrieve().onStatus(HttpStatus.TOO_MANY_REQUESTS::equals, get429StatusHander())
+                    .onStatus(HttpStatus.UNPROCESSABLE_ENTITY::equals, res -> res.bodyToMono(String.class).map(VantaUpdateException::new))
```

Comment:
`422` comes back from Vanta when it thinks a resource id isn't found (even though the resource id is one that we got back from a previous `get`) so that we can log more information like `Resource of type resource not found with 'id' identifier matching '[id]'`.